docs/ABI/Mangling.rst

```diff
@@ -583,12 +583,16 @@ mangled in to disambiguate.
   impl-function-type ::= type* 'I' FUNC-ATTRIBUTES '_'
   impl-function-type ::= type* generic-signature 'I' PSEUDO-GENERIC? FUNC-ATTRIBUTES '_'
 
-  FUNC-ATTRIBUTES ::= CALLEE-ESCAPE? CALLEE-CONVENTION FUNC-REPRESENTATION? PARAM-CONVENTION* RESULT-CONVENTION* ('z' RESULT-CONVENTION)
+  FUNC-ATTRIBUTES ::= CALLEE-ESCAPE? DIFFERENTIABILITY-KIND? CALLEE-CONVENTION FUNC-REPRESENTATION? PARAM-CONVENTION* RESULT-CONVENTION* ('z' RESULT-CONVENTION)
```

Comment:
Minor: missing `SWIFT_ENABLE_TENSORFLOW` markers for changes to `docs/ABI/Mangling.rst` added in this patch and in https://github.com/apple/swift/pull/25804. I'm not sure how to add comments in `.rst` files though.

Reply:
Comments start with `..` I believe. We have `.. SWIFT_ENABLE_TENSORFLOW` in SIL.rst.

Reply:
Thanks! Done in https://github.com/apple/swift/pull/26597.